src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -396,13 +396,13 @@ private function getClasses(array $entrypointProperties, array $classes): array
                 [
                     '@type' => 'hydra:SupportedProperty',
                     'hydra:property' => [
-                        '@id' => '#ConstraintViolationList/violation',
-                        '@type' => 'rdf:Property',
-                        'rdfs:label' => 'violation',
+                        '@id' => '#ConstraintViolationList/violations',
+                        '@type' => 'hydra:Link',
```

Comment:
Are you sure about this change? We include the document so it's not a link (a link should refers to an IRI, not to an embedded document IIRC).